### PR TITLE
Add a cool signup form

### PR DESCRIPTION
**Pull request by yucombinator**
It is very cool.
Also is a form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,7 +73,34 @@ <h2>While you wait...</h2>
...
+        <label for="cats"><span><i>Optional:</i> Thought on the movie Cats (2019)?</span></label>
+      </div>
+
+
+      <p style="text-align: center"><a class="button" href="https://forms.gle/mPmo9hbThWfjDeuz5" id="apply">Apply</a></p>
+
+      <p style="text-align: center">...and then check out the <a href="https://www.facebook.com/events/316698152296365/">Facebook event!</a></p>
+
+      </form>
+    </section>
```
Comment (davepagurek) [inline index.html]: I think this `</section>` doesn't have a matching opening tag, was there supposed to be one?

File: index.html [hunk 2]
```diff
@@ -73,7 +73,34 @@ <h2>While you wait...</h2>
...
+
+      <div class="item">
+        <input type="text" name="cats" class="question" id="cats" required autocomplete="off" />
+        <label for="cats"><span><i>Optional:</i> Thought on the movie Cats (2019)?</span></label>
+      </div>
+
+
+      <p style="text-align: center"><a class="button" href="https://forms.gle/mPmo9hbThWfjDeuz5" id="apply">Apply</a></p>
+
+      <p style="text-align: center">...and then check out the <a href="https://www.facebook.com/events/316698152296365/">Facebook event!</a></p>
```
Comment (yucombinator) [inline index.html]: TODO: this is the wrong link

File: index.html (hunk 1 above)
Comment (yucombinator) [reply index.html]: oops its gone now

**Review by davepagurek — APPROVED** [review]
nice, thanks for adding this!

**Pull request merged**